Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -1379,9 +1382,13 @@ function vtkOpenGLTexture(publicAPI, model) {
           data
         );
       }
-      if (model.oglNorm16Ext && dataType === VtkDataTypes.SHORT) {
+      if (
+        model.oglNorm16Ext &&
+        !model.useHalfFloat &&
+        dataType === VtkDataTypes.UNSIGNED_SHORT
+      ) {
         for (let c = 0; c < numComps; ++c) {
-          model.volumeInfo.scale[c] = 32767.0;
+          model.volumeInfo.scale[c] = 65535.0;
```

Comment:
I think this scale info was a bug for `unsigned_short`

Reply:
This makes sense to me.